### PR TITLE
PLAT-137559: Update Slider

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated Slider according to the new design requirements. 

### Links
[//]: # (Related issues, references)
PLAT-137559

### Comments

### CODE REVIEW

File: styles/variables-base.less [hunk 1]
```diff
@@ -583,7 +583,11 @@
...
 @agate-slider-vertical-margin: 0 @agate-component-spacing;
 @agate-slider-vertical-padding: (@agate-slider-height / 2) ((@agate-slider-height - @agate-slider-track-height) / 2);
+@agate-slider-vertical-max-height: auto;
+@agate-slider-vertical-min-height: 120px;
 @agate-slider-height: @agate-button-height;
+@agate-slider-horizontal-max-width: auto;
+@agate-slider-horizontal-min-width: auto;
 @agate-slider-width: @agate-slider-height;
 @agate-slider-track-height: 12px;
 @agate-slider-track-width: @agate-slider-track-height;
```
Comment (adrian-cocoara-lgp) [inline styles/variables-base.less:593]: Please order alphabetically

File: styles/variables-silicon.less [hunk 2]
```diff
@@ -301,6 +301,10 @@
 
 // Slider
 // ---------------------------------------
+@agate-slider-horizontal-max-width: 1179px;
+@agate-slider-horizontal-min-width: 420px;
+@agate-slider-vertical-max-height: @agate-slider-horizontal-max-width;
+@agate-slider-vertical-min-height: @agate-slider-horizontal-min-width;
 @agate-slider-width: @agate-slider-knob-height;
 @agate-slider-knob-height: 30px;
 @agate-slider-horizontal-anchorpoint-width: (@agate-slider-knob-height * @agate-slider-knob-resting-state-scale - 1px); // just a touch under the resting state scale, to avoid subpixel rendering overflow
```
Comment (adrian-cocoara-lgp) [inline styles/variables-silicon.less:310]: Please order alphabetically

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: styles/variables-silicon.less [hunk 3]
```diff
@@ -301,20 +301,24 @@
...
-@agate-slider-vertical-anchorpoint-height: (@agate-slider-knob-height * @agate-slider-knob-resting-state-scale - 1px); // just a touch under the resting state scale, to avoid subpixel rendering overflow
-@agate-slider-knob-border-radius: (@agate-slider-knob-height / 2);
-@agate-slider-knob-resting-state-scale: 0.9;
-@agate-slider-active-knob-height: 60px;
 @agate-slider-active-knob-border-radius: (@agate-slider-active-knob-height / 2);
+@agate-slider-active-knob-height: 60px;
 @agate-slider-active-knob-transform: @agate-translate-center scale(@agate-slider-knob-resting-state-scale);
 @agate-slider-border-radius: 0;
+@agate-slider-horizontal-anchorpoint-width: (@agate-slider-knob-height * @agate-slider-knob-resting-state-scale - 1px); // just a touch under the resting state scale, to avoid subpixel rendering overflow
+@agate-slider-horizontal-max-width: 1179px;
```
Comment (MikyungKim) [inline styles/variables-silicon.less:309]: I think we don't need to specify max-width from the component level. Let's just add min-width only.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**